packages/@react-native-windows/cli/src/runWindows/utils/deploy.ts

```diff
@@ -181,7 +181,7 @@ function getWindowsStoreAppUtils(options: RunWindowsOptions) {
     'WindowsStoreAppUtils.ps1',
   );
   execSync(
-    `${powershell} -NoProfile Unblock-File "${windowsStoreAppUtilsPath}"`,
+    `${powershell} -NoProfile Unblock-File "'${windowsStoreAppUtilsPath}'"`,
```

Comment:
the double quotes that already exist should have addressed this, do you know why they don't? do we need both the double and single quote or would the single quote work by itself?

Reply:
From my reading, the double quotes alone works to escape spaces when using `cd`, but when you are running a powershell script, wrapping the script path in double quotes is a correct syntax, but it will not escape spaces. If you want to escape spaces you need to have single quotes. Looks like just single quotes also works here; the double quotes don't affect behavior either way.